### PR TITLE
Fragments navigation icon and screen tracking names

### PR DESCRIPTION
**Pull request by vycius**
@mjurkus I would like your opinion about `BaseFragment` builder.

Fixes regression with fragment navigation introduced in https://github.com/vilnius/tvarkau-vilniu/pull/201

### CODE REVIEW

File: app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt [hunk 1]
```diff
@@ -58,19 +71,42 @@ abstract class BaseFragment : Fragment(), OnBackPressed {
     override fun onResume() {
         super.onResume()
 
-        analytics.trackOpenFragment(activity, javaClass.simpleName)
+        screen?.run {
+            if (titleRes != 0) {
```
Comment (mjurkus) [inline app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:75]: When can it be 0?

File: app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt [hunk 2]
```diff
@@ -58,19 +71,42 @@ abstract class BaseFragment : Fragment(), OnBackPressed {
...
+                activity.setTitle(titleRes)
+            }
+
+            trackingScreenName.emptyToNull()?.let {
```
Comment (mjurkus) [inline app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:79]: Here too?

Comment (vycius) [reply app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:79]: You have a choice if you want to track fragment or not. If name is empty string tracking is disabled.

File: app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt (hunk 1 above)
Comment (vycius) [reply app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:75]: It's annotation default. It means that fragment shouldn't set title.

Comment (mjurkus) [reply app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:75]: I don't see any cases where you're using empty title, though.

Comment (vycius) [reply app/src/main/java/lt/vilnius/tvarkau/fragments/BaseFragment.kt:75]: For example in NewReport fragment title is set dynamically by report type.

**Review by mjurkus — APPROVED** [review]

**Pull request merged**